Dockerfile.dev

```diff
@@ -0,0 +1,25 @@
+# to run locally:
+# docker build -f Dockerfile.dev -t <some-awesome-app-name> .
+# docker run -v $(pwd):/app -p 8080:8080 --rm <some-awesome-app-name>
+
+FROM node:20.11.1
+
+WORKDIR /app
+
+# Copy package.json and package-lock.json for installing dependencies
+COPY package*.json ./
+
+# Install project dependencies
+RUN npm install
+
+# Copy the rest of your app's source code from your host to your image filesystem.
+COPY . .
+
+# Install project dependencies
+# RUN npm i
+
+# Vue CLI serves on port 8080 by default, expose that port
+# EXPOSE 8080
+
+# Command to run the app using npm
+# CMD ["npm", "run", "dev-server"]
```

Comment:
Thanks @aaronkanzer.  Are these commented commands required?  If not, I would suggest removing them.

```suggestion
```

Reply:
Whoops -- good catch, those commands were meant to not be commented out